docs/content/reference/install-configuration/tls/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Traefik Proxy  HTTPS & TLS Overview |Traefik Docs"
+description: "Traefik supports HTTPS & TLS, which concerns roughly two parts of the configuration: routers, and the TLS connection. Read the documentation to learn more."
+---
+
+# Introduction
+
+## Configuring TLS Certificates
+
+When configuring TLS in Traefik, there are two main components:
+
+- The routers that handle TLS traffic,
+- The TLS certificates that are served.
+
+Routers in Traefik are responsible for directing incoming requests to the appropriate services.
+To enable TLS on a router, you specify the TLS configuration within the router’s definition.
+This setup tells Traefik that the router should handle requests using the TLS protocol, thus ensuring that the data is encrypted.
+
+Providing TLS certificates is essential for establishing secure connections.
+Traefik supports several methods for managing these certificates:
+
+- **User-defined certificates** - Provided using the file provider or Kubernetes Secrets in the Traefik dynamic configuration.
+
+    ```yaml tab="File (YAML)"
+    # Dynamic configuration
+
+    tls:
+      certificates:
+        - certFile: /path/to/domain.cert
+          keyFile: /path/to/domain.key
+        - certFile: /path/to/other-domain.cert
+          keyFile: /path/to/other-domain.key
+    ```
+
+    ```toml tab="File (TOML)"
+    # Dynamic configuration
```

Comment:
@nmengin @sheddy-traefik is it normal to document dynamic config options in the Install Configuration?

Reply:
Yes. Technically, there shouldn't be the use of dynamic or static configuration wording anymore